Assets/Scripts/AbstractPlayerControllerCommand.cs

```diff
@@ -14,29 +14,25 @@ public class MoveForward : AbstractPlayerControllerCommand
 {
     public override void Execute(GameObject aCharacter, CharacterController aController)
     {
-        float hMove = 0.0f;
-        float vMove = 5.0f;
-        float speed = 1.0f;
+        float hMove = 5.0f;
+        // float vMove = 5.0f;
```

Comment:
Perhaps delete here and in further places since we no longer use it for the player's movement?